src/Patches/GhostHints.cs

```diff
@@ -332,23 +332,33 @@ public HintGhost() { }
             },
             { "Library Hall", new List<HintGhost>() {
                 new HintGhost("Hint Ghost Library Hall 1", "Library Hall", new Vector3(113.4677f, 12.4772f, -69.9041f), new Quaternion(0f, 0.8486f, 0f, -0.529f), NPC.NPCAnimState.SIT, $"hehlO... R yoo frehnds wi% #uh lIbrArEuhn?"),
                 new HintGhost("Hint Ghost Library Hall 2", "Library Hall", new Vector3(154.6941f, 12.5768f, -39.4832f), new Quaternion(0f, 0.9531f, 0f, -0.3025f), NPC.NPCAnimState.SIT, $"I kahnt rEj mI kawfE... wAr R #Oz wawki^ rObawt tAbuhlz\nwehn yoo nEd #ehm?") }
             },
-            { "Library Exterior", new List<HintGhost>() {
-                new HintGhost("Hint Ghost Library Exterior 1", "Library Exterior", new Vector3(27.4042f, 44f, 9.9574f), new Quaternion(0f, 0.9912f, 0f, 0.1326f), NPC.NPCAnimState.SIT, $"wuht R yoo dooi^ awl #uh wA uhp hEr?---... wuht ahm I dooi^ awl #uh wA uhp hEr?\nriturni^ ahn Ovurdoo lIbrArE bouk...", interactRadiusOverride: 4.2f, cameraYOverride: 310),
-                new HintGhost("Hint Ghost Library Exterior 2", "Library Exterior", new Vector3(-8.2745f, 44.0645f, -36.2473f), new Quaternion(0.018f, -0.2001f, -0.0037f, -0.9796f), NPC.NPCAnimState.SIT, $"#uh lIbrArEuhn iz uhp #Ar... $oud bE A tehkstbouk\nfIt for yoo #O!") }
+            { "Library Rotunda", new List<HintGhost>() {
+                new HintGhost("Hint Ghost Library Rotunda 1", "Library Rotunda", new Vector3(-25.3f, -8.2f, 38.7f), new Quaternion(0f, .9343f, 0f, -.3566f), NPC.NPCAnimState.PRAY, $"wI did #uh lIbrArEuhn Evehn bild awl #Ez lahdurz?\nkahnt hE flI?"),
+                new HintGhost("Hint Ghost Library Rotunda 2", "Library Rotunda", new Vector3(-8.5f, -7f, 1.5f), new Quaternion(0f, .9973f, 0f, .0738f), NPC.NPCAnimState.SIT, $"#is yoosd too bE uh bAbE slorm rAstrahk bahk in %uh dA,\ngood tImz...") }
             },
-            { "Dusty", new List<HintGhost>() {
-                new HintGhost("Hint Ghost Dusty", "Dusty", new Vector3(64.3287f, 26.058f, 46.5875f), new Quaternion(0f, 0.9282f, 0f, -0.372f), NPC.NPCAnimState.IDLE, $"R yoo goi^ too klEn uhp #is mehs? wuht A rilEf!", cameraYOverride: 50), }
+            { "Library Lab", new List<HintGhost>() {
+                new HintGhost("Hint Ghost Library Lab 1", "Library Lab", new Vector3(139.3969f, 93.5073f, -74.8239f), new Quaternion(0f, 1f, 0f, 0.0089f), NPC.NPCAnimState.SIT, $"Im #uh lIbrArEuhn!---... juhst kidi^, I kahnt rEd."),
+                new HintGhost("Hint Ghost Library Lab 2", "Library Lab", new Vector3(131.3964f, 107.4616f, -37.7488f), new Quaternion(0f, 0.7272f, 0f, 0.6864f), NPC.NPCAnimState.FISHING, $"awl #is rEsurj ahnd #A stil dOnt nO wuht A\ndE pahd iz..."),
+                new HintGhost("Hint Ghost Library Lab 3", "Library Lab", new Vector3(145.05f, 87f, -111.35f), new Quaternion(0f, .8829f, 0f, -.4695f), NPC.NPCAnimState.SIT, $"O, yoo wawnt too nO wuht Im rEdi^? itz kawld\nfawks bEhAvyur: intrO too yahpi^ ehnd %EvurE.") }
```

Comment:
```suggestion
                new HintGhost("Hint Ghost Library Lab 3", "Library Lab", new Vector3(145.05f, 87f, -111.35f), new Quaternion(0f, .8829f, 0f, -.4695f), NPC.NPCAnimState.SIT, $"O, yoo wawnt too nO wuht Im rEdi^? its kawld\nfawks bihAvyur: intrO too yahpi^ ahnd %EvurE.") }
```
50/50 on if the behavior is correct or not, I'll leave it up to you

Reply:
Probably the capital E there -- it's more of a proper title, so it gets the more proper E sound